module.jai

```diff
@@ -642,6 +647,7 @@ str_from_numeric :: (data: *u8, len: int, row: int, col: int) -> string, success
 
 #import "Basic";
 #import "Pool";
+#import "String";
```

Comment:
Why is this necessary now?

Reply:
Just for parse_int for:
```jai
            rows_affected := to_string(PQcmdTuples(query_res));
            rows_affected_count := parse_int(*rows_affected);
```

If there's another way to parse I'll swap it over and remove this.

Reply:
You can just use `string_to_int` from `Basic`.